### DISCUSSION TITLE
Menu Item

### DISCUSSION THREAD

**Discussion by daddasoft** [mantinedev/mantine#2875] [Q&A] [answered]
i want to ask if that a bug or just how it should be i use remix i have a user button witch is a menu target  and  other menu items one of theme i want to  make the item as button as following :

```jsx
<Menu.Item as="button" type="submit">
Logout 
</Menu.Item>
```
but the button rendered as button of type  "button" I'am wondering why

**Comment by rtivital** [comment, marked as answer]
I'll add an option to override type attribute in one of the next releases

**Reply by daddasoft** [reply]
thanks  for help i'm looking forward to it
